src/proto/srv/srv.proto

```diff
@@ -89,3 +89,19 @@ message CheckReportReq {
 message CheckReportResp {
 	int32 status = 1; // DAOS error code.
 }
+
+message ListPoolsReq
+{
+	bool includeAll = 1; // Include all pools in response, regardless of state
```

Comment:
[Minor] Should be written as `include_all`, which would give `includeAll` in Go and `include_all` in C. (See https://protobuf.dev/programming-guides/style/.)

Reply:
Thanks for catching this. I've updated it